### ISSUE TITLE
Java class name high-lighting

### ISSUE THREAD

**Issue by philiprbrenan** [geany/geany#691]
![class name highlighting](https://cloud.githubusercontent.com/assets/7548378/10420820/f96eac1e-7093-11e5-9114-3d96c5095e95.png)
<b>Geany</b> very helpfully high-lights Java class names, which I find very useful.  However, if I use a curly D as the first letter of the name, which is valid Java, it doesn't seem to high-light the class name.

**Comment by codebrainz** [comment]
You are evil for using ambiguous Unicode characters in an identifier, even if it's valid code :)

**Comment by esotericpig** [comment]
I was looking through some old bugs, and this one seems to be fine now and can be closed. @codebrainz 

![Screenshot from 2021-06-03](https://user-images.githubusercontent.com/16524392/120540519-5f7b2880-c41b-11eb-8814-6471ee34be89.png)

**Comment by elextr** [comment]
@esotericpig doesn't work here.

Note: the symbol in the symbols tab is `isplay` no curly D and the display is not __bolded__ (not highlight as the OP called it).

**Comment by esotericpig** [comment]
> @esotericpig doesn't work here.
> 
> Note: the symbol in the symbols tab is `isplay` no curly D and the display is not **bolded** (not highlight as the OP called it).

Oh, you're right, my bad. I had to save it to a file (not just a file buffer with setting the file type manually) for the bold and symbols to show up, not sure if that is also a bug.

**Comment by elextr** [comment]
>  I had to save it to a file (not just a file buffer with setting the file type manually) for the bold and symbols to show up, not sure if that is also a bug.

No, thats how it works, the symbols need a filename since thats the key they are stored under.
